driver-mapping/src/test/java/com/datastax/driver/mapping/MapperTest.java

```diff
@@ -416,7 +417,7 @@ public Thread apply(Void v) {
         executeFunctionAndTestForException(u, mapper, new Function<Void, Thread>() {
             @Override
             public Thread apply(Void v) {
-                mapper.save(u, null);
```

Comment:
You are absolutely right, I'm sorry, I missed that detail. 
But in this case, let's use something else than `null` because `null` will eventually throw a `NullPointerException` – for example, if `checkNotInEventLoop()` succeeds because the callback is being executed on the calling thread – and make the test fail.